### PR TITLE
adding taxonomy-meta to header

### PR DESCRIPTION
**Pull request by lahirue**

### CODE REVIEW

File: modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js [hunk 1]
```diff
@@ -20,6 +20,7 @@
 var resources = function () {
     return {
         css:['common.css','sidepanel.css', 'left-navigation.css', 'font-wso2.css', 'bootstrap-dialog.min.css', 'typeahead.css'],
-        js:['sidepanel.js','greg-subscriptions-api.js','bootstrap-dialog.min.js', 'typeahead.bundle.min.js', 'typeahead.js']
+        js:['sidepanel.js','greg-subscriptions-api.js','bootstrap-dialog.min.js', 'typeahead.bundle.min.js', 'typeahead.js'],
```
Comment (splinter) [inline modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js]: Spacing between the : [

File: modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js [hunk 2]
```diff
@@ -20,6 +20,7 @@
...
+        code: ['taxonomy-tenant-meta.hbs']
```
Comment (splinter) [inline modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js]: Spacing between the : and [

Comment (lahirue) [inline modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js]: Thank you Sameera for mentioning it,
This has been fixed with wso2/carbon-store#834

File: modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js (hunk 1 above)
Comment (lahirue) [inline modules/es-extensions/store/extensions/app/greg-store-defaults/themes/store/helpers/header.js]: Thank you Sameera for mentioning it,
This has been fixed with wso2/carbon-store#834

**Pull request merged**